handler/http/server.go

```diff
@@ -371,10 +373,10 @@ func (h *DrandHandler) getRand(ctx context.Context, chainHash []byte, info *chai
 	ctx, cancel := context.WithTimeout(ctx, h.timeout)
 	defer cancel()
 	resp, err := bh.client.Get(ctx, round)
-
 	if err != nil {
 		return nil, err
 	}
+	h.log.Debugw("Happy path request fulfilled", "round", round, "resp", resp.GetRound())
```

Comment:
seems weird to print both `round` and `GetRound()` - should the latter just be the whole `resp`?

Reply:
`round` is the requested input, `resp.GetRound` is the round of the "output". I'll remove this log, it's too verbose anyway